src/components/flat-table/__snapshots__/flat-table.spec.js.snap

```diff
@@ -50,7 +51,8 @@ exports[`FlatTable when rendered with proper table data should have expected str
 }
 
 .c0 .c7,
-.c0 .c2 .c5 {
+.c0 .c2 .c5,
+.c0 .c2 .sc-jzJRlG {
```

Comment:
I think that we should not have dynamic classes rendered in snapshots

Reply:
Yeah this is what I was talking about offline, I haven't been able to work out why I'm not getting a placeholder className here so might just rewrite the test to avoid snapshots if I have no luck

Reply:
The issue seems to be related to the order in which imported `styled-components` are declared in a `style` file, I have changed the order and the dynamic class has been replaced by the placeholder